src/main/java/dev/task/dndquest/service/impl/PlayCharacterClassServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package dev.task.dndquest.service.impl;
+
+import dev.task.dndquest.exception.DBException;
+import dev.task.dndquest.model.entity.PlayCharacterClass;
+import dev.task.dndquest.repository.PlayCharacterClassRepository;
+import dev.task.dndquest.service.PlayCharacterClassService;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class PlayCharacterClassServiceImpl implements PlayCharacterClassService {
+    private final PlayCharacterClassRepository repository;
+
+    @Override
+    public PlayCharacterClass findByName(String name) {
+        return repository.findByName(name)
+               .orElseThrow(() -> new DBException("no such character class in database"));
```

Comment:
DBException is a very poor name. U will have plenty. Use specific naming - CharNotFoundException

Reply:
ok